runtimes/runtimes/operational-telemetry/operational-telemetry-service.ts

```diff
@@ -0,0 +1,85 @@
+import { PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics'
+import { AWSMetricExporter } from './aws-metric-exporter'
+import { OperationalTelemetry } from './operational-telemetry'
+import opentelemetry, { Attributes } from '@opentelemetry/api'
+import { NodeSDK } from '@opentelemetry/sdk-node'
+import { Resource } from '@opentelemetry/resources'
+import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions/*'
+import { randomUUID } from 'crypto'
+
+export class OperationalTelemetryService implements OperationalTelemetry {
+    private customResource: Record<string, any> = {}
+    private static instance: OperationalTelemetryService
+    private SCOPE_NAME = 'language-server-runtimes'
```

Comment:
This needs to scale to servers in the future.

Reply:
You're right, I will include it in the metrics schema.